include/aikido/distance/NominalConfigurationRanker.hpp

```diff
@@ -0,0 +1,35 @@
+#ifndef AIKIDO_DISTANCE_NOMINALCONFIGURATIONRANKER_HPP_
+#define AIKIDO_DISTANCE_NOMINALCONFIGURATIONRANKER_HPP_
+
+#include "aikido/distance/ConfigurationRanker.hpp"
+
+namespace aikido {
+namespace distance {
+
+class NominalConfigurationRanker : public ConfigurationRanker
+{
+public:
+  /// Constructor
+  ///
+  /// \param[in] metaSkeletonStateSpace Statespace of the skeleton.
+  /// \param[in] metaskeleton Metaskeleton of the robot.
+  /// \param[in] nominalConfiguration Nominal Configuration. The current
```

Comment:
Nit: `Nominal Configuration` -> `Nominal configuration`.

Can we also provide a `nullptr` default?